### PR TITLE
feat(desktop/launcher): handle new window links with wexond

### PR DESCRIPTION
**Pull request by jvnm-dev**

### CODE REVIEW

File: src/launcher/index.ts [hunk 1]
```diff
@@ -56,6 +56,38 @@ async function main() {
     log.error(error)
   })
 
+  const openBrowserWindow = async (
+    flowrStore: Store<IFlowrStore>,
```
Comment (lmancel) [inline src/launcher/index.ts:60]: Shadowed variable name

File: src/launcher/index.ts [hunk 2]
```diff
@@ -154,9 +164,22 @@ async function main() {
...
       })
+
+      flowrWindow.webContents.on('new-window', async (event, url) => {
+        event.preventDefault()
+
+        await openBrowserWindow(store, event, {
+          clearBrowsingDataAtClose: false,
+          openUrl: url,
+          maxTab : 1,
+          enableVirtualKeyboard: true,
```
Comment (lmancel) [inline src/launcher/index.ts]: this property is set from the store: https://github.com/taktik/flowr-desktop/pull/33/files#diff-47ee3c25cc8e4af787f0e42cbbd44f6f34fc8a5986419eee8efc5b3adcaef37dR69

File: src/launcher/index.ts [hunk 3]
```diff
@@ -154,9 +164,22 @@ async function main() {
...
         flowrWindow = null
       })
+
+      flowrWindow.webContents.on('new-window', async (event, url) => {
+        event.preventDefault()
+
+        await openBrowserWindow(store, event, {
+          clearBrowsingDataAtClose: false,
+          openUrl: url,
+          maxTab : 1,
```
Comment (lmancel) [inline src/launcher/index.ts]: Let's set it to 0 instead, to limit further navigation
```suggestion
          maxTab : 0,
```

**Review by lmancel — CHANGES_REQUESTED** [review]

File: src/launcher/index.ts (hunk 1 above)
Comment (jvnm-dev) [reply src/launcher/index.ts:60]: `flowrStore` is declared only after this function in the `onReady`

**Review by lmancel — APPROVED** [review]

**Pull request merged**